lib/loadbalanceproperties.py

```diff
@@ -77,7 +89,8 @@ def getAppropriateLoadBalancer(self):
         if self.placements == '':
             ld = LoadBalanceProperties.CONNECTION_MANAGER_MAP.get(self.SIMPLE_LB)
             if ld == None:
-                ld = ClusterAwareLoadBalancer.getInstance()
+                # print("Refresh Interval", self.EQUALS)
+                ld = ClusterAwareLoadBalancer.getInstance(300)
```

Comment:
Please check if this needs to pass 300 or the provided value of `self.refreshInterval`.

Reply:
I still think we should provide `self.refreshInterval` instead of 300 here. Can you confirm?